packages/opentelemetry-cloud-monitoring-exporter/test/transform.test.ts

```diff
@@ -115,51 +153,58 @@ describe('transform', () => {
           },
         ],
       };
-
       assert.deepStrictEqual(
         createTimeSeries(
           data,
           {labels: {}, type: ''},
           'workload.googleapis.com'
         ),
-        [
```

Comment:
Diff butchered this change. This test now contain a reference to the expected data structure above. The rest of the file is a new test.